test/bdd/did_exchange_steps.go

```diff
@@ -39,6 +39,20 @@ func (d *DIDExchangeSteps) createInvitation(inviterAgentID string) error {
 	return nil
 }
 
+func (d *DIDExchangeSteps) createInvitationWithDID(inviterAgentID string) error {
+	invitation, err := d.bddContext.DIDExchangeClients[inviterAgentID].CreateInvitationWithDID(inviterAgentID, d.bddContext.PublicDIDs[inviterAgentID].ID)
+	if err != nil {
+		return fmt.Errorf("failed to create invitation: %w", err)
```

Comment:
Just a thought that I am posting everywhere I see wrapping error strings that have "failed to" in them :). https://github.com/uber-go/guide/blob/master/style.md#error-wrapping

Reply:
Will fix